### ISSUE TITLE
Questions about the "age_seconds" property in particle point o.history

### ISSUE THREAD

**Issue by SuiYu2002** [OpenDrift/opendrift#1444]
O.hiory can check all the records, now I have two questions to ask
1. Why does the "age_seconds" attribute in the first record not start at 0
![7c073b88bbeff692dc561e163fddfe2e](https://github.com/user-attachments/assets/7f74da61-9b77-4f93-8ad1-05edb46d359f)
2. Why are the "age_seconds" attributes of the last two records the same![7c073b88bbeff692dc561e163fddfe2e]
![8c2edf300d54d9e5a26d29e605d2fc9e](https://github.com/user-attachments/assets/95e4a991-cea8-4f4d-87af-39ab52ed0e7e)

**Comment by knutfrode** [comment]
You are right this was a bug, which is now fixed: https://github.com/OpenDrift/opendrift/commit/b8cc6763d3cc5ad92c85f44f9837fcd9991facd3
Thank you for spotting and reporting.
